<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="RemoveMapElementLine"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="FindMapElementLine"/>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="FindUnorderedMapElement">
        <property name="text">
         <string>Find Element</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="FindSetElement">
        <property name="text">
         <string>Find Element</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="RemoveSetElementLine"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="FindMapElement">
        <property name="text">
         <string>Find Element</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="RemoveUnorderedMapElement">
        <property name="text">
         <string>Remove element</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="RemoveMapElement">
        <property name="text">
         <string>Remove ELement</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="FindSetElementLine"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="RemoveSetElement">
        <property name="text">
         <string>Remove Element</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="FindUnorderedMapElementLine"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="AddMapElement">
        <property name="text">
         <string>Add Element to Map</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="AddSetElement">
        <property name="text">
         <string>Add Element to Set</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QPushButton" name="AddUnorderedMapElement">
        <property name="text">
         <string>Add Element to Unordered Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="RemoveUnorderedMapElementLine"/>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QListWidget" name="UnorderedMap"/>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QTreeWidget" name="Set">
        <column>
         <property name="text">
          <string>Set</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTreeWidget" name="Map">
        <column>
         <property name="text">
          <string>Map</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
